client/modules/workspace/src/SelectModal/SelectModal.tsx

```diff
@@ -109,83 +98,141 @@ const getScheme = (storageSystem: TTapisSystem): string => {
-          label={systemLabel}
-        />
-      ),
-      dataIndex: 'name',
-      ellipsis: true,
-
-      render: (data, record) =>
-        record.format === 'folder' ? (
+        <div>
           <Button
-            style={{
-              marginLeft: '3rem',
-              textAlign: 'center',
-              display: 'flex',
-              alignItems: 'center',
-            }}
-            onClick={() => navCallback(encodeURIComponent(record.path))}
             type="link"
+            onClick={() =>
+              navCallback(
+                getBackPath(
+                  path,
+                  searchTerm,
+                  clearSearchTerm,
+                  selectedSystem,
+                  user
+                )
+              )
+            }
           >
-            <i
-              role="none"
-              className="fa fa-folder-o"
-              style={{ color: '#333333', marginRight: '8px' }}
-            ></i>
-            {data}
+            <LeftOutlined />
+            Back
           </Button>
-        ) : (
-          <span
-            style={{
-              marginLeft: '3rem',
-              textAlign: 'center',
-              display: 'flex',
-              alignItems: 'center',
-            }}
-          >
+        </div>
+      ),
+      dataIndex: 'name',
+      ellipsis: true,
+
+      render: (data, record, index) => {
+        const isFolder = record.format === 'folder';
+        const marginLeft = index === 0 ? '3rem' : '6rem';
+        const commonStyle = {
+          marginLeft,
+          textAlign: 'center' as const,
+          display: 'flex',
+          alignItems: 'center',
+        };
+        const iconClassName = isFolder ? 'fa fa-folder-o' : 'fa fa-file-o';
+
+        if (isFolder && index > 0) {
+          return (
+            <Button
+              style={commonStyle}
+              onClick={() => navCallback(encodeURIComponent(record.path))}
+              type="link"
+            >
+              <i
+                role="none"
+                className={iconClassName}
+                style={{ color: '#333333', marginRight: '8px' }}
+              ></i>
+              {data}
+            </Button>
+          );
+        }
+
+        return (
+          <span style={commonStyle}>
             <i
               role="none"
-              className="fa fa-file-o"
+              className={iconClassName}
               style={{ color: '#333333', marginRight: '8px' }}
             ></i>
             {data}
           </span>
-        ),
+        );
+      },
     },
     {
       dataIndex: 'path',
       align: 'end',
       title: '',
-      render: (_, record) => {
-        const shouldRenderSelectButton =
+      render: (_, record, index) => {
+        const selectionModeAllowed =
           (record.type === 'dir' && selectionMode === 'directory') ||
           (record.type === 'file' && selectionMode === 'file') ||
           selectionMode === 'both';
+        const isNotRoot =
```

Comment:
Can we still allow project system roots to be added?

<img width="620" alt="Screenshot 2024-07-10 at 3 19 58 PM" src="https://github.com/DesignSafe-CI/portal/assets/20326896/72f23466-fbb9-48a0-bb98-81ffeacd1c2b">

Reply:
Fixed this. And tested root selection in projects and ensured it is still not showing in other systems.

![Screenshot 2024-07-11 at 8 07 29 AM](https://github.com/DesignSafe-CI/portal/assets/2568355/c7331855-baac-4fb6-99c3-7eda620bf36c)